hack/make/test-install-script

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# This script is used for testing install.sh and that it works for
+# each of component of our apt and yum repos
+set -e
+
+test_install_script(){
+	# these are equivalent to main, testing, experimental components
+	# in the repos, but its the url that will do the conversion
+	components=( get test experimental )
+
+	for component in "${components[@]}"; do
+		# change url to specific component for testing
+		local test_url=https://${component}.docker.com
+		local script=/tmp/install.sh
```

Comment:
Shouldn't this use something like `mktemp install-XXXXXXXXXX.sh` instead of a hard-coded path? (and then quotes around the places we use it, ie `sed ... > "$script"`, `-v "$script":/install.sh`, etc)